### PR TITLE
add instagram button to partner site headers, update zazzle to store.…

### PR DESCRIPTION
**Pull request by tanyaparker**
…code.org

**Before**
image

**After**
image

### CODE REVIEW

File: pegasus/sites.v3/br.code.org/views/header.haml [hunk 1]
```diff
@@ -13,8 +13,10 @@
         %img{:src=>"/images/twitter_purple.png"}
       %a{:href=>"http://codeorg.tumblr.com/", :title=>'Tumblr', :target=>"_blank", :style=>"text-decoration:none"}
         %img{:src=>"/images/tumblr_purple.png"}
-      %a{:href=>"http://zazzle.com/codeorg", :title=>'Zazzle', :target=>"_blank", :style=>"text-decoration:none"}
-        %img{:src=>"/images/zazzle_purple.png"}
+      %a{:href=>"https://www.instagram.com/codeorg/", :title=>'Instagram', :target=>"_blank", :style=>"text-decoration:none"}
```
Comment (ashercodeorg) [inline pegasus/sites.v3/br.code.org/views/header.haml]: nit: Any reason not to remove the trailing slash in the `href` be removed? Save a byte. :) Also below (for the store) and with other languages.

**Comment by ashercodeorg** [comment]
Presumably the shirt icon is generic and not specific to (copyrighted by) zazzle? Do any of these changes touch eyes tests? If so, CC the next DOTD. If not, I'll encourage creating eyes test(s) for a partner site.

Presumably the change in login button is because of a change in login state or something, not related to this change? Why am I not seeing a "Hi, Tanya" if you logged in for the "after" state.

Barring those comments, LGTM.

**Comment by ashercodeorg** [comment]
Also, I'm seeing a bit of funniness in the CSS. It seems like the rightmost icon is a bit smaller.

image

(from https://tr.code.org/ when logged in)

**Comment by tanyaparker** [comment]
Yup! Shirt icon seems generic. Couldn't find anything on their site using that. Was thinking about replacing it with a font awesome icon but they don't have one.

Locally didn't have the login button because I was just running pegasus-server instead of dashboard-server.

Ugh. Sign in button is off by 1 at the top when signed out and off by 1 at the bottom when signed in. :P Filing to fix later.

Going to open a separate PR to add a bunch of eye tests (including teacher dashboard, code studio from previous teacher announcement change).

**Pull request merged**